parcel.go

```diff
@@ -12,49 +13,114 @@ func NewParcelStore(db *sql.DB) ParcelStore {
 	return ParcelStore{db: db}
 }
 
+// Add добавляет строку с информацией о посылке
 func (s ParcelStore) Add(p Parcel) (int, error) {
-	// реализуйте добавление строки в таблицу parcel, используйте данные из переменной p
-
-	// верните идентификатор последней добавленной записи
-	return 0, nil
+	res, err := s.db.Exec("INSERT INTO parcel (client, status, address, created_at) VALUES (:client, :status, :address, :created_at)",
+		sql.Named("client", p.Client),
+		sql.Named("status", p.Status),
+		sql.Named("address", p.Address),
+		sql.Named("created_at", p.Created_At))
+	if err != nil {
+		return 0, fmt.Errorf("failed to add parcel info: %w", err)
+	}
+
+	id, err := res.LastInsertId()
+	if err != nil {
+		return 0, fmt.Errorf("failed to get last inserted id: %w", err)
+	}
+
+	return int(id), nil
 }
 
+// Get реализует чтение одной строки из таблицы parcel по заданному number
 func (s ParcelStore) Get(number int) (Parcel, error) {
-	// реализуйте чтение строки по заданному number
-	// здесь из таблицы должна вернуться только одна строка
-
-	// заполните объект Parcel данными из таблицы
 	p := Parcel{}
-
+	err := s.db.QueryRow("SELECT number, client,status,address,created_at FROM parcel WHERE number = ?", number).Scan(
+		&p.Number, &p.Client, &p.Status, &p.Address, &p.Created_At)
+	if err != nil {
+		return p, fmt.Errorf("failed to query parcel info with number %d: %w", number, err)
```

Comment:
Лучше с ошибкой возвращать пустые данные
```suggestion
		return Parcel{}, fmt.Errorf("failed to query parcel info with number %d: %w", number, err)
```